src/components/PlanPage/PlanPage.js

```diff
@@ -1,37 +1,35 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import Loader from 'react-loader';
 
-import ActivitiesListFilter from '../ActivitiesListFilter';
 import Week from '../Week';
 import { startSetActivities } from '../../store/activities/actions';
 
-class PlanPage extends React.Component {
-  static propTypes = {
-    onStartSetActivities: PropTypes.func.isRequired,
-  };
+const PlanPage = ({ onStartSetActivities }) => {
+  const [isDataLoaded, setIsDataLoaded] = useState(false);
 
-  state = {
-    isLoaded: false,
+  const asyncOnStartSetActivities = () => {
```

Comment:
I don't understand why are you doing this :P it's very, very weird. Can you tell what is the target of this operation ?